lxd/container_lxc.go

```diff
@@ -6012,8 +6012,15 @@ func (c *containerLXC) idmapsetFromConfig(k string) (*shared.IdmapSet, error) {
 }
 
 func (c *containerLXC) NextIdmapSet() (*shared.IdmapSet, error) {
```

Comment:
You can probably now remove the additional check performed in `init()` whether `c.daemon.IdmapSet == nil`. But that's just minor.